### DISCUSSION TITLE
Референсы к новому дизайну сайта

### DISCUSSION THREAD

**Discussion by Darolga** [atls/kino-reporter#11] [General]
Подборка в [фигме](https://www.figma.com/file/2FXKL1VVln1GGXYhL4w1hI/%D0%9A%D0%B8%D0%BD%D0%BE%D1%80%D0%B5%D0%BF%D0%BE%D1%80%D1%82%D0%B5%D1%80-%2F-%D0%A0%D0%B5%D1%84%D1%8B?node-id=0%3A1) 
[Подборка скринов на диске](https://drive.google.com/drive/folders/10gGAHKuNXSPPId_Rw7sPxas15YExhLw1?usp=sharing)

**Comment by Darolga** [comment]
Подборка сайтов
https://www.chicagomag.com/
https://www.msk.kp.ru/
https://pen-online.com/
https://www.forbes.com/?sh=4e0b618b2254
https://www.thinkglobalhealth.org/
https://cuprum.media/
https://www.behance.net/gallery/123814309/The-Conversation-news-website-redesign-concept
https://www.behance.net/gallery/129802429/CNN-News-portal
https://www.kinopoisk.ru/media/article/4005394/
https://www.ivi.tv/movies/all
https://megafon.tv/
https://www.hollywoodreporter.com/

**Comment by AlexeyGovorukhin** [comment]
Из приведенных референсов Чикаго самый симпатичный, на мой взгляд)
Голливуд репортер у нас раньше был на thr.ru
КП - вырви глаз
иви и мегафон - кинотеатральный каталог 
кинопоиск  (и другие российские киносайты) мы точно не будем использовать как референс

Важное при выборе вёрстки: 
1) как я писал в соседней ветке, более 90% посетителей приходят на конкретный материал из поиска, социальной сети, агрегаторов. Доля переходов (с внутренних или внешних) на главную страницу - не более 3%. Поэтому задача удержать посетителей на странице материала - через подгрузку следующего материала, вставку сниппетов, гиперссылок на релевантыне материалы  и т.п.
2) более 75% посетителей - с мобильных устройств, поэтому крайне важно, чтобы сайт был привлекательным и удобным для мобильных
3) судя по тепловой карте, мало кто из посетителей прокручивают ниже 2-го экрана, поэтому крайне важно зацепить уже на первом и сделать качественный и удобный поиск
